docs/ops/opset1.md

```diff
@@ -58,8 +58,8 @@ declared in `namespace opset1`.
 * [LogicalOr](logical/LogicalOr_1.md)
 * [LogicalXor](logical/LogicalXor_1.md)
 * [LRN](normalization/LRN_1.md)
-* [LSTMCell](sequence/LSTMCell_1.md)
-* [LSTMSequence](sequence/LSTMSequence_1.md)
+* [LSTMCell](sequence/LSTMCell_4.md)
+* [LSTMSequence](sequence/LSTMSequence_5.md)
```

Comment:
Why do you update `opset1.md`? There was no introduced `LSTMCell_4` at that time.
The same comment about changes in `opset2`, `opset3`

Reply:
the current specification is aligned with LSTMSequence v5 implementation
LSTMSequence v1 contains unsupported features e.g. ctor with Peepholes, input_forget and gates order, I'm not so sure that we actually need to describe these things and introduce LSTMSequence v1 spec
not so sure which option is less confusing

Reply:
Ivan, let us discuss this tomorrow offline.

Reply:
@jane-intel, I think this one not quite correct changes in our documentation. It brings some distraction here. Please advice.